zhaquirks/ikea/twobtnremote.py

```diff
@@ -195,3 +195,63 @@ class IkeaTradfriRemote2BtnZLL(CustomDevice):
     }
 
     device_automation_triggers = IkeaTradfriRemote2Btn.device_automation_triggers.copy()
+
+class IkeaRodretRemote2Btn(CustomDevice):
+    """Custom device representing IKEA of Sweden RODRET remote control."""
```

Comment:
Better description of the device then its 2 more upcoming Shortcut and OpenClose and the first is CSA certed for some weeks ago.

Reply:
SOMRIG shortcut button Aka. E2213

https://csa-iot.org/csa_product/somrig-shortcut-button/

OpenCloce is coming then IKEA have using all produces hardware they have in storage at the factorys.

Reply:
I've not seen a SOMRIG yet -- happy to have a go at adding when I get one. Let me know how to handle different firmware variants and I'll add that later too.

Reply:
> I've not seen a SOMRIG yet -- happy to have a go at adding when I get one. Let me know how to handle different firmware variants and I'll add that later too.

Hey. I have a somrig Remote and its not recognizing any button press in the log, will your quirk help? What do i need to change in order to make it work for somrig? 
I am a very beginner, so keeop that in mind. Thank you soo much..

Reply:
I s´was getting little parts today https://github.com/zigpy/zigpy/discussions/660#discussioncomment-7775801 and all is working without quirks but Sommerig (Suller like in Eng) is using matter ZCL  commands on one manufacture cluster like Symfonisk 2 is doing so need little work foe getting it working but im sure its coming very son.

Reply:
Hardware is identical only the printing on it is different and the firmware is different and sending commands on one different cluster that is not standard and need one quirk for fixing it,

Reply:
Any news? I would also donate you something. 
Cheers

Reply:
Im on one 2  weeks trip and have only laptop with and no connection to my test networks so i cant do any testings but if some one like coding and testing i can helping with little knowledge that i was taking with ;-)